src/interfaces/addons.ts

```diff
@@ -1,12 +1,12 @@
 import { ITagType } from './tags';
 
 export interface IAddon {
-    id: number;
     provider: string;
-    description: string;
-    enabled: boolean;
+    parameters: Record<string, any>;
```

Comment:
Could we use `Record<string, unknown>` here without too much work?

Reply:
Significant enough to maybe do it with next set of TS refactors